From 30fb836a1a534bf321c538a36369b12d6fe88283 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:15 -0400
Subject: [redhat] kabi: Adding symbol find_first_bit (lib/find_next_bit.c)

Message-id: <87ce79e5df2ac19b3e51044bf4d7262014fd4c8e.1410510773.git.poros@redhat.com>
Patchwork-id: 95927
O-Subject: [PATCH RHEL7.1 075/233] kabi: Adding symbol find_first_bit (lib/find_next_bit.c)
Bugzilla: 1077335
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    This API being used for bit-map manipulations in our Flow-Structures.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_first_bit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_first_bit b/redhat/kabi/kabi-module/kabi_x86_64/find_first_bit
new file mode 100644
index 0000000..48332f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_first_bit
@@ -0,0 +1,2 @@
+#1-
+0xb352177e find_first_bit vmlinux EXPORT_SYMBOL
-- 
1.7.1